.github/workflows/integration.yml

```
@@ -204,3 +201,45 @@ jobs:
         if: always()
         continue-on-error: true
         run: docker logs --since "${{ env.START_TIME }}" sockethub-ergo-1
+
+  contract:
+    runs-on: ubuntu-latest
+
+    steps:
+      - run: sudo chown -R $USER:$USER ${{ github.workspace }}
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)